### PR TITLE
Added load event tracking documentation

### PR DESCRIPTION
**Pull request by terrymun**
This commit also reformats how tracking payloads are being presented on the page, for easy reading and standardized formatting. I have also rearranged some text around to make them flow better.

### CODE REVIEW

File: pages/integration--events-tracked-by-google-analytics-and-tag-manager.md [hunk 1]
```diff
@@ -4,79 +4,324 @@ title: Events tracked by Google Analytics / Tag Manager
...
-{ 'event': 'virtualPageview', 'virtualPageURL': flipbookurl + '?Page=' + pagenumber }
+window.ga(tracker.get('name') + '.send', {
+	hitType: 'event',
+	eventCategory: 'iPaper events',
+	eventAction: [ACTION],
+	eventLabel: [LABEL]
+});
 ```
 
-We'll also track various **Events** across our flipbooks. Take a look at the list below for a full list of events.
```
Comment (terrymun) [inline pages/integration--events-tracked-by-google-analytics-and-tag-manager.md]: Even though this table is more readable in markdown, it's not once it is generated: the reason being that the label texts are difficult to read and are not monospaced. I have therefore taken the liberty to convert them into multiline code for better readability.

File: pages/integration--events-tracked-by-google-analytics-and-tag-manager.md [hunk 2]
```diff
@@ -97,7 +342,19 @@ window.dataLayer.push({
 ```
 
 ### Extras
-Aside from the DataLayers above we also push `PageElementClicked`, but it's only triggered for  the **internal** & **external** linktype.
+
+The following features have also been enabled for GTM:
```
Comment (cyberzed) [inline pages/integration--events-tracked-by-google-analytics-and-tag-manager.md]: We recently remove the whitelist...can you nuke it as well...or should we merge and then I do it as I have a working jekyll setup?

Comment (terrymun) [reply pages/integration--events-tracked-by-google-analytics-and-tag-manager.md]: Whitelist has been removed :)

**Comment by terrymun** [comment]
@cyberzed We should probably merge this soon :) it's been hanging around for a bit.

**Pull request merged**